qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -25,6 +25,23 @@
 				"{#for r in recordsList.take(3)}"
 			],
 			"url": "https://quarkus.io/guides/qute-reference#collections"
+		},
+		{
+			"signature": "or(self: T, arg : java.lang.Object) : boolean",
+			"description": "Returns the default value if the base object is null, empty or not found and the base Object otherwise",
+			"sample": [
+				"{item.price.or(5)}",
+				"{name ?: 'elvis'}"
+			],
+			"url": "https://quarkus.io/guides/qute-reference#built-in-resolvers"
+		},
+		{
+			"signature": "ifTruthy(self : T, arg : java.lang.Object) : T",

```

Comment:
replace `self `with `base`. Please note that  the correct signature should be 

`"ifTruthy(base : java.lang.Object, arg : T) : T"`, but today method parameters are not supported, so it's not possible to write this signature. I will do that in an another PR once https://github.com/redhat-developer/quarkus-ls/pull/504 will be merged.

Reply:
Please write java test with ifTruthy